docs/parameter_description.rst

```diff
@@ -0,0 +1,158 @@
+must correspond to a child object of the selected type (Image are child of Dataset and Project, \
+Well are child of Screen and Plate, ...). The option "*<all (from selected)>*" is used to apply the script \
+on the selected object and its children (dataset and images in the case of a project selected).
+
+
+File Annotation
+---------------
+Parameter for the import from a CSV file. Provided from the "*Choose File*" selection menu, \
+the file will be temporarily uploaded to OMERO for the duration of the script.
+
+The items listed in the CSV are matched to the found target objects. If an item of the CSV correspond to no \
+target object, the item is skipped (as such, target objects coming from different parent objects can be \
+annotated from a single CSV).
+
+The matching of items in the CSV to target objects is done either via IDs or names (IDs have priority over names, \
+see "*Target ID colname*"" and "*Target name colname*"").
+
+IDs of File Annotation already attached to the object can be used instead of "*Choose File*":
+
+.. image:: images/annotation_13_file_annotation_id.png
+
+Namespace
+---------
+In the case of key-value pairs import, the namespace assigned to the created key-value pairs.
+
+For the three other scripts, used to restrict the selection of key-value pairs to that namespace.
+
+When no namespace is provided, matches the namespace ``openmicroscopy.org/omero/client/mapAnnotation``, which \
+makes the key-value pairs editable in OMERO.web (the only namespace that allows that).
+
+Multiple namespace can be provided as a comma separated list.
+
+All namespace can be selected using the * character.
+
+Old Namespace
+-------------
+For the namespace conversion script. List of old namespace (comma separated) to convert to a different namespace (unique). \
+This can be used to group key-value pairs into a single annotations (all found key-value pairs will be grouped to a single\
+key-value pair).
+
+New Namespace
+-------------
+For the namespace conversion script. The new namespace to assign to the key-value pairs found on the target \
+object having the "*Old Namespace*".
+
+Create new and merge
+--------------------
+For the namespace conversion script. When ticked, the script will create a single new annotation regrouping all the old \
+annotation it found, instead of only updating the namespace of each annotations. The same parameter can be given to \
+"*Old Namespace*" and "*New Namespace*" just to regroup the annotations.
+
+CSV separator
+-------------
+For the import script, the separator used in the CSV to separate elements. By default \
+set to 'guess' and will attempt to detect automatically the separator among ``,`` ``;`` ``TAB`` ``|``.
+
+For the export script, the separator used in the generated CSV to separate elements. We \
+recommend the use of TAB.
+
+Include parent container names
+------------------------------
+For the export script. Check the box if you want to include in the CSV output \
```

Comment:
aha, here you are ``checking`` the box. I will suggest this also in another place, where you are using the expression ``ticking``.